apps/console-lite/src/app/components/portfolio/orders/use-column-definitions.tsx

```diff
@@ -0,0 +1,287 @@
+              prefix + addDecimal(value, data.market.positionDecimalPlaces)
+            );
+          }
+          return '-';
+        },
+      },
+      {
+        colId: 'type',
+        field: 'type',
+        width: 80,
+        valueFormatter: ({
+          value,
+        }: ValueFormatterParams & {
+          value?: Orders_party_ordersConnection_edges_node['type'];
+        }) => OrderTypeMapping[value as OrderType],
+      },
+      {
+        colId: 'status',
+        field: 'status',
+        cellClass: 'text-center font-mono !flex h-full items-center',
+        valueFormatter: ({
+          value,
+          data,
+        }: ValueFormatterParams & {
+          value?: StatusKey;
+        }) => {
+          if (value && data && data.market) {
+            if (value === OrderStatus.STATUS_REJECTED) {
+              return `${OrderStatusMapping[value as StatusKey]}: ${
+                data.rejectionReason &&
+                OrderRejectionReasonMapping[
+                  data.rejectionReason as RejectReasonKey
+                ]
+              }`;
+            }
+            return OrderStatusMapping[value as StatusKey] as string;
+          }
+          return '-';
+        },
+      },
+      {
+        colId: 'filled',
+        headerName: t('Filled'),
+        headerClass: 'uppercase',
+        field: 'remaining',
+        cellClass: 'font-mono text-center !flex h-full items-center',
+        width: 80,
+        valueFormatter: ({
+          data,
+          value,
+        }: ValueFormatterParams & {
+          value?: Orders_party_ordersConnection_edges_node['remaining'];
+        }) => {
+          if (value && data && data.market) {
+            const dps = data.market.positionDecimalPlaces;
+            const size = new BigNumber(data.size);
+            const remaining = new BigNumber(value);
+            const fills = size.minus(remaining);
+            return `${addDecimal(fills.toString(), dps)}/${addDecimal(
+              size.toString(),
+              dps
+            )}`;
+          }
+          return '-';
+        },
+      },
+      {
+        colId: 'price',
+        field: 'price',
+        type: 'rightAligned',
+        width: 125,
+        headerClass: 'uppercase text-right',
+        cellClass: 'font-mono text-right !flex h-full items-center',
+        valueFormatter: ({
+          value,
+          data,
+        }: ValueFormatterParams & {
+          value?: Orders_party_ordersConnection_edges_node['price'];
+        }) => {
+          if (
+            value === undefined ||
+            !data ||
+            !data.market ||
+            data.type === OrderType.TYPE_MARKET
+          ) {
+            return '-';
+          }
+          return addDecimal(value, data.market.decimalPlaces);
+        },
+      },
+      {
+        colId: 'timeInForce',
+        field: 'timeInForce',
+        minWidth: 120,
+        valueFormatter: ({
+          value,
+          data,
+        }: ValueFormatterParams & {
```

Comment:
yeah, right, I think it's just a beginning of portfolio works. Currently is only roughly a copy of lib version. When we will have designed views, in next tasks this can be changed completely